wodles/aws/aws_s3.py

```diff
@@ -3101,8 +3118,14 @@ def get_script_arguments():
                         help='URL for the VPC endpoint to use to obtain the STS token.')
     parser.add_argument('-se', '--service_endpoint', type=str, dest='service_endpoint', default=None,
                         help='URL for the VPC endpoint to use to obtain the logs.')
+    parser.add_argument('-ds', '--duration-seconds', type=int, dest='role_session_duration', default=None,
```

Comment:
Replace `-ds` and `--duration-seconds` with a more descriptive parameter name, such as `--iam_role_duration`.